ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-nightly-x86.yaml

```diff
@@ -53,6 +53,22 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: rosa-aws-sts
   timeout: 5h0m0s
+- as: data-path-9nodes
+  cron: 0 13 * * 3
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.xlarge
+      CONFIG: config/standard-classic-rosa.yml
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.15"
+      REPLICAS: "9"
+    test:
+    - chain: openshift-qe-data-path-tests
```

Comment:
`- ref: openshift-qe-perfscale-aws-data-path-sg`

Need to add this to ensure we can run network tests using hostNetwork.

Reply:
You have the ordering wrong. the path-sg needs to be first.

Reply:
Ack, updated ptal